src/entity/Menu.java

```diff
@@ -0,0 +1,74 @@
+package entity;
+
+import java.io.Serializable;
+
+public class Menu implements Serializable, Comparable<Menu> {
```

Comment:
Why did you implement Serializable?

Reply:
Because we need convert to a byte-stream and used in DB(csv file)